internal/core/docker_host_test.go

```diff
@@ -462,10 +462,17 @@ func TestExtractDockerSocketFromClient(t *testing.T) {
 
 		mockCallbackCheck(t, testCallbackCheckError)
 
+		// check that panic is raised
+		defer func() {
+			if r := recover(); r == nil {
+				t.Errorf("The code did not panic")
+			}
+		}()
+
```

Comment:
suggestion: require has a Panics helper, reset of the test will need removing.
```suggestion
		require.Panics(t, func() {
			extractDockerSocketFromClient(ctx, mockCli{OS: "Ubuntu"})
		})
```